.github/workflows/trigger.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021, Proofcraft Pty Ltd
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+# Trigger docker deploy action in seL4/ci-actions on C parser changes
+
+name: Tigger
+
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - 'tools/c-parser/**'
+  # for testing:
+  workflow_dispatch:
+
+jobs:
+  trigger:
+    name: CParser Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.SSRG_BAMBOO_REPO }}
+          repository: seL4/ci-actions
+          event-type: cparser-deploy
```

Comment:
I couldn't find a workflow to receive this event. I guess you still need to add that?

Reply:
Never mind. I should have looked in the [pull requests](https://github.com/seL4/ci-actions/pull/106)!

Reply:
I also just added that after this one, so there was a race condition :-)